compiler/rustc_codegen_llvm/src/va_arg.rs

```diff
@@ -52,7 +52,7 @@ fn emit_direct_ptr_va_arg(
     let next = bx.inbounds_gep(addr, &[full_direct_size]);
     bx.store(next, va_list_addr, bx.tcx().data_layout.pointer_align.abi);
 
-    if size.bytes() < slot_size.bytes() && &*bx.tcx().sess.target.target.target_endian == "big" {
+    if size.bytes() < slot_size.bytes() && bx.tcx().sess.target.target.target_endian.is_big() {
```

Comment:
Why not just `*bx.tcx().sess.target.target.target_endian == Endian::Big`? Do we need a helper function to compare value?

Reply:
duplicate https://github.com/rust-lang/rust/pull/77604#discussion_r500578080 .